lib/__tests__/tickplate.js

```diff
@@ -0,0 +1,48 @@
+/* eslint-disable no-template-curly-in-string */
```

Comment:
That's how it is usually done in Jest, but it is not a requirement. If I recall correctly, you may as well name the directory as you wish and just name your test files as `*.test.js` or something like that.